client/src/components/TextField/tests/TextField-story.js

```diff
@@ -41,4 +51,21 @@ storiesOf('Admin/TextField', module)
       description="Field description"
       message={{ value: 'Validation message', type: 'error' }}
     />
-  ));
+  ))
+  .add(
+    'With tip',
+    withNotes(
+      withTipNotes
+    )(
+      () => (
+        <div style={{ margin: '5em', width: '30em' }}>
+          <TextField
+            {...props}
+            value="Default text"
+            description="Field description"
+            tip={{ icon: 'lamp', importance: 'high', content: 'Additional information about the field' }}
```

Comment:
Could have use some knobs here, so the user can try out different values.

Reply:
I've applied knobs across the whole `TextField` story, along with the new `Tip` story.